required/latex-lab/latex-lab-math.dtx

```diff
@@ -205,11 +205,8 @@
 %  The actual tagging is in done through the sockets
 %   \begin{itemize}
 %    \item |tagsupport/math/inline/formula/begin|
-%     This socket takes the math as argument and its code
+%     This socket takes the math as second argument and its code
 %     should output it for typesetting.
```

Comment:
Nesting tagging sockets  seems a bit strange. Not looked at the full file to see where they are used elsewhere.

Reply:
math tagging is complicated and for better organization sockets are nested. the inner sockets could get different names but which one? Simply removing the tagsupport string doesn't look right, I would like to keep an indication that this is tagging code.

Reply:
so do you expect them to get different plugs beside the normal ("do tagging" and "don't do tagging")? Otherwise I don't understand the argument.

Reply:
you mean the inner sockets? Yes could be. But it also avoids code duplication for code that is needed both in inline and display math (but I'm not claiming that it has to be done this way for eternity, once math tagging has settled down one should certainly review that).